papertty/papertty.py

```diff
@@ -685,6 +688,14 @@ def terminal(settings, vcsa, font, fontsize, noclear, nocursor, cursor, sleep, t
         print("--nocursor is deprecated. Use --cursor=none instead")
         settings.args['cursor'] = None
 
+    if vcom:
+        vcom = int(vcom)
+        if vcom <= 0:
+            print("VCOM should be a positive number. It will be converted automatically. eg. For a value of -1.46V, set VCOM to 1460")
+            sys.exit(1)
+        else:
+            settings.args['vcom'] = vcom
```

Comment:
`else` is not really needed here:
```suggestion
    if vcom:
        vcom = int(vcom)
        if vcom <= 0:
            print("VCOM should be a positive number. It will be converted automatically. eg. For a value of -1.46V, set VCOM to 1460")
            sys.exit(1)
        settings.args['vcom'] = vcom
```